GmailUtils.gs

```diff
@@ -476,8 +476,8 @@ function formatEmails_(emails) {
  * @param {string} class
  * @return {boolean}
  */
-function isa_(obj, class) {
-  return typeof obj == 'object' && typeof obj.constructor == 'undefined' && obj.toString() == class;
+function isa_(obj, className) {
+  return typeof obj == 'object' && (typeof obj.constructor == 'undefined' || typeof obj.constructor == 'function') && obj.toString() == className;
```

Comment:
V8 is now supporting class and constructors. I've tested this update and it works for me